app/src/main/java/io/homeassistant/companion/android/launch/my/MyActivity.kt

```diff
@@ -0,0 +1,60 @@
+package io.homeassistant.companion.android.launch.my
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import io.homeassistant.companion.android.BaseActivity
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.webview.WebViewActivity
+import kotlinx.android.synthetic.main.activity_webview.webview
+
+class MyActivity : BaseActivity() {
+    companion object {
+        val EXTRA_URI = "EXTRA_URI"
+
+        fun newInstance(context: Context, uri: Uri): Intent {
+            return Intent(context, MyActivity::class.java).apply {
+                putExtra(EXTRA_URI, uri.toString())
+            }
+        }
+    }
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_my)
+
+        if (Intent.ACTION_VIEW == intent?.action && intent.data != null) {
+            if (intent.data?.getQueryParameter("mobile")?.equals("1") == true) {
+                finish()
+                return
+            }
+            val newUri = intent.data!!.buildUpon().appendQueryParameter("mobile", "1").build()
+
+            WebView.setWebContentsDebuggingEnabled(true)
```

Comment:
Should we only do this on debug builds?  I think thats what we did in WebViewActivity

Reply:
Doh.... Fixed